src/Components/__tests__/ProgressBar.spec.js

```diff
@@ -6,7 +6,6 @@ describe("<ProgressBar />", () => {
   let progressBar;
   const props = {
     currentSectionName: "conv",
-    currentSection: 0,
     totalSections: 7,
```

Comment:
we should also add some tests that are directly testing the progress bar sections. Maybe we  can add tests that makes sure the current section never exceeds the total number of sections? Also, how are we testing the UI of the progress bar sections? For example, when currentSection is 4 and total sections are 8, what should the progress bar look like?

Reply:
I added a UI test for the current and active bar sections, but since the current section is a local var. I'm not sure how I can access it through the test, unless I rewrote the component as a class component. If the incorrect number is passed in the progress bar still renders. I need to make this a TODO for this component.

Reply:
I think you should be able to mock the current section variable, but I'm not 100% sure. The tests look good for now and we can add a TODO to possibly add more in the future.